src/vs/workbench/contrib/terminalContrib/chatAgentTools/browser/executeStrategy/executeStrategy.ts

```diff
@@ -96,6 +96,23 @@ export function detectsCommonPromptPattern(cursorLine: string): IPromptDetection
 	return { detected: false, reason: `No common prompt pattern found in last line: "${cursorLine}"` };
 }
 
+
+// PowerShell-style multi-option line (supports [?] Help and optional default suffix) ending in whitespace
+const PS_CONFIRM_RE = /\s*(?:\[[^\]]\]\s+[^\[]+\s*)+(?:\(default is\s+"[^"]+"\):)?\s+$/;
+
+// Bracketed/parenthesized yes/no pairs at end of line: (y/n), [Y/n], (yes/no), [no/yes]
+const YN_PAIRED_RE = /(?:\(|\[)\s*(?:y(?:es)?\s*\/\s*n(?:o)?|n(?:o)?\s*\/\s*y(?:es)?)\s*(?:\]|\))\s+$/i;
+
+// Same as YN_PAIRED_RE but allows a preceding '?' or ':' and optional wrappers e.g. "Continue? (y/n)" or "Overwrite: [yes/no]"
+const YN_AFTER_PUNCT_RE = /[?:]\s*(?:\(|\[)?\s*y(?:es)?\s*\/\s*n(?:o)?\s*(?:\]|\))?\s+$/i;
```

Comment:
Another important format:

<img width="641" height="262" alt="image" src="https://github.com/user-attachments/assets/7d8986f7-a098-4578-9fff-fe2c043b342e" />

Looking at the code `yo` generator would be good to support too.